schemas/test/en/test_supplementary_data.json

```diff
@@ -1544,6 +1544,204 @@
                     ]
                 }
             ]
+        },
+        {
```

Comment:
Perhaps this is just me, but when I was going through the scenarios, it made it slightly clearer to test when I added a section summary to these new sections. As otherwise when going back to view the section via 'view answers' you can only see the interstitial page, which makes it confusing as to how many list items there are.